app.js

```diff
@@ -89,6 +105,22 @@ app.get('/api/v1/projects', (request, response) => {
     });
 });
 
+app.post('/api/v1/projects', (request, response) => {
+  const project = request.body
```

Comment:
Same feedback as the previous POST endpoint. Overall nice clean POST endpoints hitting the appropriate status codes!